moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py

```diff
@@ -181,7 +181,7 @@
 
    # Get template folder location
    try:
-      plugin_gen_dir = roslib.packages.get_pkg_dir(plugin_gen_pkg)
+      plugin_gen_dir = roslib.packages.get_pkg_dir(plugin_gen_pkg) + '/ikfast_kinematics_plugin'
```

Comment:
Please `os.path.join(..)` for this.

Reply:
It would also be nice to make the `ikfast_kinematics_plugin` string constant an actual constant (as far as that is possible in Python).

Put it next to the `plugin_gen_pkg` constant at the top of the file.

Reply:
Done.